fileserver/routes.py

```diff
@@ -313,10 +313,12 @@ def get_session_version():
     platform = request.args.get("platform")
 
     if platform not in ("desktop", "android", "ios"):
-        app.logger.warn("Invalid session platform '{}'".format(platform))
+        app.logger.warning("Invalid session platform '{}'".format(platform))
         return error_resp(http.NOT_FOUND)
     project = "session-foundation/session-" + platform
 
+    release_channel = request.args.get("release_channel") if request.args.get("release_channel") else 'latest'
```

Comment:
Perhaps `stable` rather than `latest`?

Reply:
Actually, looking at how this runs, that probably requires a fair amount of extra work so I'm fine leaving it as is.

Reply:
So I used "latest" because that is term used by our electron compiler ([electron-builder builder](https://www.electron.build/)). But Audric has already asked for the UI to use "stable" so I am happy to make the change here too.

Will update